pkgs/development/tools/rust/cargo-flash/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, rustPlatform, pkg-config, rustfmt, libusb1 }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-flash";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "probe-rs";
+    repo = pname;
+    rev = "v" + version; 
```

Comment:
Suggested change:
```suggestion
    rev = "v${version}";
```

Reply:
IIRC I once got the feedback to do the swap the other way round, as string interpolation is supposed to be more expensive than string concatenation. Is that wrong? I'm eager to know any rational to promote either the one or the other style.

Reply:
I don't think it has much to do with speed, rather consistency and just the code being prettier. Reading the rev takes almost no time anyway, that doesn't really matter.

There's 9 instances of `rev = "v" + version;` being used throughout the codebase, versus the 1957 instances of `rev = "v${version}";`. For consistency's sake thats much better.

Reply:
Whatever :laughing:

Reply:
maybe also this? :D